upi-india/issuer-switch/adapter/README.md

```diff
@@ -0,0 +1,12 @@
+# Google Cloud Issuer Switch - Bank Adapter APIs
+
+Bank Adapter developers can use the proto files to generate an OpenAPI spec and
+subsequently use that for their development of bank adapter APIs.
+They can also use the Postman collection for testing their implementation.
+
+Please note that the data used for fields like IFSC, account numbers, mobile numbers, etc. in the 
+postman collection is purely for testing purposes and does not represent any real values.
+
+## License
+
+Apache 2.0; see [`LICENSE`](LICENSE) for details.
```

Comment:
We will need to update this link to refer to root directory LICENSE file. I think this should work /LICENSE
https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes#relative-links-and-image-paths-in-readme-files